### PR TITLE
Update chargeitem create

### PR DESCRIPTION
**Pull request by krvicky31**
Description
----
Update fhir.cerner.com for ChargeItem write/create

PR Checklist
----
- [x] Screenshot(s) of changes attached before change
Screenshot 2021-11-18 at 4 08 59 PM
Screenshot 2021-11-18 at 4 09 32 PM
Screenshot 2021-11-18 at 4 10 29 PM
Screenshot 2021-11-30 at 3 33 52 PM
Screenshot 2021-11-30 at 3 34 22 PM
Screenshot 2021-11-30 at 3 34 42 PM
Screenshot 2021-11-30 at 3 35 11 PM
Screenshot 2021-11-30 at 3 35 29 PM
Screenshot 2021-11-30 at 3 35 43 PM
Screenshot 2021-11-30 at 3 36 01 PM
Screenshot 2021-11-30 at 3 36 17 PM
Screenshot 2021-11-30 at 3 36 32 PM
Screenshot 2021-11-30 at 3 36 51 PM
Screenshot 2021-11-30 at 3 37 09 PM
Screenshot 2021-11-30 at 3 40 17 PM
Screenshot 2021-11-30 at 4 00 38 PM
Screenshot 2021-11-30 at 4 01 37 PM
Screenshot 2021-11-18 at 4 15 19 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/r4/charge_item_create.yaml [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+              'subject': {
+                'reference': 'Patient/178594204'
+               }
+            }
+         ]
+       }
+      }
+    url: https://hl7.org/fhir/chargeitem-definitions.html#ChargeItem
+
+  - name: chargeItem.resourceType
```
Comment (krvicky31) [inline lib/resources/r4/charge_item_create.yaml:47]: @SantanuBanerjee  the name is in correct format or it should be only resourceType

**Comment by Nandita92** [comment]
Can you please post the screenshot of local fhir.cerner.com page for these changes

File: lib/resources/r4/charge_item_create.yaml [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+              'resource': {
+              'resourceType': 'chargeItem',
+              'status': 'unknown',
+              'subject': {
+                'reference': 'Patient/178594204'
+               }
+            }
+         ]
+       }
+      }
```
Comment (Nandita92) [inline lib/resources/r4/charge_item_create.yaml:44]: Check on indentation

File: lib/resources/r4/charge_item_create.yaml [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+          {
+            "name": "chargeItem",
+            "reason": [
+              {
+                "extension": [
+                  {
+                    "valueUnsignedInt": 1,
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+                  }
+                ],
```
Comment (Nandita92) [inline lib/resources/r4/charge_item_create.yaml:235]: If this description is for reason field, I don't think so you need to add coding and extension field for this.
Any opinion @SantanuBanerjee ?

**Comment by krvicky31** [comment]
> Can you please post the screenshot of local fhir.cerner.com page for these changes
added all the screenshots.
Screenshot 2021-11-15 at 2 16 13 PM
Screenshot 2021-11-15 at 2 16 41 PM
Screenshot 2021-11-15 at 2 17 30 PM
Screenshot 2021-11-15 at 2 17 42 PM
Screenshot 2021-11-15 at 2 18 00 PM
Screenshot 2021-11-15 at 2 18 26 PM
Screenshot 2021-11-15 at 2 18 44 PM
Screenshot 2021-11-15 at 2 19 05 PM
Screenshot 2021-11-15 at 2 19 22 PM
Screenshot 2021-11-15 at 2 19 36 PM
Screenshot 2021-11-15 at 2 19 50 PM
Screenshot 2021-11-15 at 2 20 18 PM

**Comment by krvicky31** [comment]
> 

Attached the Screenshot of the changes done

File: lib/resources/r4/charge_item_create.yaml (hunk 3 above)
Comment (krvicky31) [reply lib/resources/r4/charge_item_create.yaml:209]: [Fixed here](https://github.com/krvicky31/fhir.cerner.com/commit/155c91bc8fd4c872bcdff517e52e8bd3af56021d#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR366)

File: lib/resources/r4/charge_item_create.yaml (hunk 2 above)
Comment (krvicky31) [reply lib/resources/r4/charge_item_create.yaml]: [Fixed indentation](https://github.com/cerner/fhir.cerner.com/pull/769/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR39-R44)

**Comment by SantanuBanerjee** [comment]
I am a little confused about the branch creation @krvicky31 .
Did you have to fork the repository by any chance because ideally the branch name does not appear the way it is displaying right now in the PR.

File: lib/resources/r4/charge_item_create.yaml [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+    type: resource
+    description: The ChargeItem to be created.
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "chargeItem",
+            "resource": {
+              "resourceType": "chargeItem"
```
Comment (SantanuBanerjee) [inline lib/resources/r4/charge_item_create.yaml:36]: I think the resourceType should be ChargeItem with C capital.

**Comment by krvicky31** [comment]
> I am a little confused about the branch creation @krvicky31 . Did you have to fork the repository by any chance because ideally the branch name does not appear the way it is displaying right now in the PR.

It has been forked.

**Comment by SantanuBanerjee** [comment]
> > I am a little confused about the branch creation @krvicky31 . Did you have to fork the repository by any chance because ideally the branch name does not appear the way it is displaying right now in the PR.
> 
> It has been forked.

I think you will need to create a direct clone from the repo, rather than forking and then create a branch from main.
Forking is usually done for the repos that we do not own, like, the norbert-config repo.
I will add more comments after the new PR has been created.

**Comment by krvicky31** [comment]
Created new pr following the guidelines to contribute on fhir.cerner.com. Attached pr link https://github.com/cerner/fhir.cerner.com/pull/778

**Pull request closed without merging**